docs/setup/adding-a-git-repository.md

```diff
@@ -186,6 +186,17 @@ The following options are supported:
           fallback_to_build_date: true
     ```
 
+`enable_creation_date`{ #enable_creation_date }
+
+:   :octicons-milestone-24: Default: `false` – This option specifies whether to also add a page *Created* date at the bottom of each page.
```

Comment:
nit: `also` seems redundant (think `This option` could be removed too)

Reply:
Fixed :)